packages/playground-preview-worker/tests/index.test.ts

```diff
@@ -45,7 +45,27 @@ describe("Preview Worker", () => {
 			'"/hello?world"'
 		);
 		expect(resp.headers.get("set-cookie") ?? "").toMatchInlineSnapshot(
-			'"token=TEST_TOKEN; Domain=random-data.playground-testing.devprod.cloudflare.dev; HttpOnly; Secure; SameSite=None"'
+			`"token=${defaultUserToken}; Domain=random-data.playground-testing.devprod.cloudflare.dev; HttpOnly; Secure; SameSite=None"`
+		);
+	});
+	it("shouldn't be redirected with invalid token", async () => {
```

Comment:
NIT - sometimes there are empty lines between `it()` clauses and sometimes not. I feel we should be consistent.